x-pack/plugins/lens/public/persistence/saved_object_store.test.ts

```diff
@@ -4,19 +4,22 @@
  * you may not use this file except in compliance with the Elastic License.
  */
 
+import { SavedObjectsClientContract, SavedObjectsBulkUpdateObject } from 'kibana/public';
 import { SavedObjectIndexStore } from './saved_object_store';
 
 describe('LensStore', () => {
   function testStore(testId?: string) {
     const client = {
       create: jest.fn(() => Promise.resolve({ id: testId || 'testid' })),
-      update: jest.fn((_type: string, id: string) => Promise.resolve({ id })),
+      bulkUpdate: jest.fn(([{ id }]: SavedObjectsBulkUpdateObject[]) =>
+        Promise.resolve({ savedObjects: [{ id }, { id }] })
+      ),
```

Comment:
Instead of using `bulkUpdate`, is it more correct to use `index`? I'm basing this off the docs which say that "update" will perform a merge, but "index" rewrites the whole thing. https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update.html

Reply:
That makes a lot of sense, thanks. I would like to get a fix out without waiting for changed core apis, do you think this approach is fine as a stopgap measure while working with the platform team to get a better solution?